spec/inverted-index-helper.js

```diff
@@ -1,25 +1,55 @@
-class invertedIndexHelper {
+/**
+ * InvertedIndexHelper
+ * @class
+ */
+class InvertedIndexHelper {
+/**
+ * getToken()
+ * Method loops through a JSON file, sorts through string, removes all special characters and spaces
+ * and converts the result into an Array
+ * @param {string}
+ * @returns {Array} splitted words in alphabetical order 
+ */
   static getToken(words) {
     const filterDuplicate = [];
-    const formattedWords = words.toLowerCase()
-      .replace(/[^A-Z0-9\s]/gi, '')
+    const formattedWords = words.toString().toLowerCase()
+      .replace(/[^A-Z0-9\s+]/gi, ' ').replace(/\s+/, ' ')
```

Comment:
### Nitpick

This makes sense, but why are you sorting? Seems like unnecessary overhead.